src/components/pill-selector-dropdown/PillSelectorDropdownField.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+import * as React from 'react';
+import getProp from 'lodash/get';
+import isEqual from 'lodash/isEqual';
+
+import type { FieldProps } from 'formik';
+
+import DatalistItem from '../datalist-item';
+import PillSelectorDropdown from './PillSelectorDropdown';
+import type { Option, Pill } from './flowTypes';
+
+import './PillSelectorDropdown.scss';
+
+type Props = FieldProps & {
+    /** CSS class for the component */
+    className?: string,
+    /** If true, the input control is disabled so no more input can be made */
+    disabled?: boolean,
+    /** If true, user can add pills not included in selector options */
+    isCustomInputAllowed: boolean,
+    /** Input label */
+    label: React.Node,
+    /** Array or Immutable list with data for the selected options shown as pills */
+    options: Array<Option>,
+    /** A placeholder to show in the input when there are no pills */
+    placeholder: string,
+    /** Called to check if pill item data is valid. The `item` is passed in. */
+    validator: (text: string) => boolean,
```

Comment:
This props collection is a lot lot less than what the PillSelectorDropdown.js has. For now I want to keep it small and expand only as need arises.